src/Types/Output/NamedOutputType.hack

```diff
@@ -11,7 +11,6 @@ namespace Slack\GraphQL\Types;
 <<__ConsistentConstruct>>
 abstract class NamedOutputType extends OutputType<this::THackType, this::TCoerced> {
 
-    <<__Enforceable>>
     abstract const type THackType;
```

Comment:
Getting rid of the `__Enforceable` attribute here both didn't seem to break anything and allowed me to repurposes `ObjectType` for objects generated from shapes even if they contain reified parameters.

Reply:
Good catch, this was only needed before #29